src/classes/PMT_PaymentCreator.cls

```diff
@@ -32,7 +32,7 @@
 * @date 2011 (2.1)
 * @description Controls the creation of payment objects from Opportunity trigger
 */
-public without sharing class PMT_PaymentCreator {
+public with sharing class PMT_PaymentCreator {
 
```

Comment:
We decided it didn't need it as it only deals with payments which are
detail records of opportunities and inherit the opp's sharing rules.

On Tue, Jul 7, 2015 at 11:01 AM, David Habib <notifications@github.com>
wrote:

> In src/classes/PMT_PaymentCreator.cls
> <https://github.com/SalesforceFoundation/Cumulus/pull/1644#discussion_r34070013>
> :
>
> > @@ -32,7 +32,7 @@
> >  * @date 2011 (2.1)
> >  * @description Controls the creation of payment objects from Opportunity trigger
> >  */
> > -public without sharing class PMT_PaymentCreator {
> > +public with sharing class PMT_PaymentCreator {
> >
>
> since this class is used by the trigger, shouldn't it be empty to inherit
> the trigger's without sharing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SalesforceFoundation/Cumulus/pull/1644/files#r34070013>
> .
>